specification/cognitiveservices/data-plane/FormRecognizer/preview/v2.0/examples/CopyOperationResult.json

```diff
@@ -0,0 +1,23 @@
+{
+  "parameters": {
+    "endpoint": "{endpoint}",
+    "Content-Type": "application/json",
+    "Ocp-Apim-Subscription-Key": "{API key}",
+    "modelId": "{modelId}",
+    "resultId": "{resultId}",
+    "body": {}
+  },
+  "responses": {
+    "200": {
+      "body": {
+        "status": "succeeded",
+        "createdDateTime": "2020-01-01T00:00:00Z",
+        "lastUpdatedDateTime": "2020-01-01T00:01:00Z",
+        "copyResult": {
+          "modelId": "{modelId}",
+          "errors": []
```

Comment:
Would it be possible to provide an example of what kinds of errors could come back from a copy operation?

Reply:
Added a new error sample with `CopyOperationResultWithErrors.json`